bedrock/careers/templates/careers/locations.html

```diff
@@ -49,20 +49,22 @@ <h2 class="c-careers-section-title">
 
       <ul class="c-careers-locations-list">
         <li>
-          <a href="{{ url('mozorg.contact.spaces.berlin') }}" data-cta-type="button" data-cta-text="Berlin">
```

Comment:
**issue:** I still see some more occurrences of these URLs on the about page:

http://localhost:8000/en-US/about/